examples/build.gradle

```diff
@@ -41,6 +63,22 @@ task geoserver(type: Exec) {
     dependsOn 'geoserverSleep'
 }
 
+task geoserverCleanup(type: Exec) {
+    description 'Run and wait for GeoServer with test data'
+    commandLine 'docker', 'exec', 'geoserver', 'rm', '-r', '/mnt/geoserver_datadir/logs'
```

Comment:
Not needed? (doesn't be done by geoserverRemoveTmpDir ?) or you have right issue?

Reply:
unfortunately the `logs` dir is written with `root` user permissions, so I have to delete it via the container